Server/bkr/server/model/scheduler.py

```diff
@@ -2313,8 +2313,10 @@ def host_requires(self):
         except ValueError:
             hrs = etree.Element('hostRequires')
 
-        # If no system_type is specified then add defaults
-        if not hrs.findall('.//system_type') and not hrs.get('force'):
+        # If no system_type or system/type is specified then add default value
```

Comment:
If no system_type or system/type is not specified?

Reply:
Corrected :dolphin: